Prime.java

```diff
@@ -0,0 +1,27 @@
+import com.sun.jdi.request.ExceptionRequest;
+
+import java.util.Scanner;
+
+public class Prime {
+    public static void main(String[] args) {
+        try {
+            Scanner sc = new Scanner(System.in);
+            System.out.println("Enter a number: ");
+            int number = sc.nextInt();
+
+            if (number == 1)
```

Comment:
logic is incorrect.  please test it before creating PR.

Reply:
done